docs/source/traits_user_manual/index.rst

```diff
@@ -1,10 +1,17 @@
+====================
 Traits 6 User Manual
 ====================
 
+:Authors: David C. Morrill, Janet M. Swisher, and Enthought developers
+:Version: Document Version 4
```

Comment:
I'm not sure whether this line is meaningful or useful. Should we just drop it?

Reply:
It hasn't been updated for 12 years so I it's probably not important

Reply:
Yep, let's remove this.